requirements/edx/edx-private.txt

```diff
@@ -36,3 +36,6 @@ ubcpi-xblock==0.5.0
 # Vector Drawing and ActiveTable XBlocks (Davidson)
 -e git+https://github.com/open-craft/xblock-vectordraw.git@ded76fc8f6c99ba4949ed57a7bf62a1f12e44683#egg=xblock-vectordraw
 -e git+https://github.com/open-craft/xblock-activetable.git@1786a2c8fc06ca3ecfac1972464048982bd486ff#egg=xblock-activetable
+
+# McKinsey Academy Diagnostic Feedback Xblock
+-e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@e87e0bc5bc2dda94d2d358a63759bffa2b0cba3c#egg=xblock-diagnostic-feedback
```

Comment:
@cahrens i have removed -e from private.txt. is this change good? CC @bradenmacdonald